src/validation.cpp

```diff
@@ -3663,6 +3673,10 @@ void BlockManager::FindFilesToPrune(std::set<int>& setFilesToPrune, uint64_t nPr
            nLastBlockWeCanPrune, count);
 }
 
+void BlockManager::UpdatePruneBlocker(std::string name, const CBlockIndex* block) {
```

Comment:
In 5d23ddf5ff3bd1e5d4f6b85440d9ad0c59bb9994 "Add prune blockers to BlockManager"

Since this has `EXCLUSIVE_LOCKS_REQUIRED`, I think it should also have `AssertLockHeld` in here too.

Reply:
Right, fixed!